pudl_scrapers/items.py

```diff
@@ -69,34 +69,13 @@ def __repr__(self):
         return "Ferc714('%s')" % self["save_path"]
 
 
-class EipInfrastructure(DataFile):
-    """The EipInfrastructure excel file."""
-
-    def __repr__(self):
-        return "EipInfrastructure('%s')" % self["save_path"]
-
-
 class LblnIsoQueues(DataFile):
```

Comment:
Should this stick around? If so, it should be LBNL (Lawrence Berkeley National Labs) not LBLN

Reply:
good catch -- it should go